hackclub.com.yaml

```diff
@@ -2954,3 +2954,7 @@ shipment-viewer:
   - ttl: 600
     type: CNAME
     value: cname.vercel-dns.com.
+for-reviewers:
+   - ttl: 600
+    type: CNAME
+    value: souptik-samanta.github.io/final-for-arcade/index.html.
```

Comment:
DNS records can't have paths (`/final-for-arcade/index.html`)

```suggestion
    value: souptik-samanta.github.io.
```

Reply:
Hey but if i change to souptik-samanta.github.io then this thing is showing
what should i do?

![image](https://github.com/user-attachments/assets/6cd2f12a-45b7-4e0d-a316-567be7c6cf99)

Reply:
You need to configure github pages https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/about-custom-domains-and-github-pages#using-a-subdomain-for-your-github-pages-site